src/components/DriverCard.jsx

```diff
@@ -0,0 +1,25 @@
+
+const DriverCard = ({ name, rating, img, car }) => {
+  return (
+    <div className="driver-card">
+      <div className="driver-card-img">
+        <img src={img} alt={name} />
+      </div>
+      <div className="driver-card-info">
+        <h2>{name}</h2>
+        <div className="driver-card-rating">
+          {Array.from({ length: 5 }, (_, index) => (
```

Comment:
Julia, here, since you received the rating, you previously built a component `Rating.jsx` that you could've used instead of repeating the process.